File: src/Rectangle.java
public class Rectangle {
    public int width = 0;
    public int height = 0;
    public Point origin;

    public Rectangle(int w, int h) {
        origin = new Point(0, 0);
        width = w;
        height = h;
    }

    // a method for moving the rectangle
    public void move(int x, int y) {
        origin.x = x;
        origin.y = y;
    }

    // a method for computing the area of the rectangle
    public int getArea() {
        return width * height;
    }
}

File: src/rectangle.py
class Rectangle:
    def __init__(self, height, width):
        self.h = height
        self.w = width
    def perim(self):
        return (2 * self.h) + (2 * self.w)
    def area(self):
        return self.h * self.w
    def isSquare(self):
        return (self.h == self.w)